packages/@okta/vuepress-site/docs/reference/ie-limitations/index.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Okta Identity Engine Limitations
+excerpt: The Okta Identity Engine introduces a lot of changes to the Okta platform. Some of these changes result in a lack of support for previously available features.
+---
+
+## Okta Identity Engine Limitations
+
+<ApiLifecycle access="ie" /><br>
+<ApiLifecycle access="Limited GA" />
+
+The Okta Identity Engine introduces a lot of changes to the Okta platform. Some of these changes result in a lack of support for previously available features. Additionally, some of these changes result in Okta Identity Engine features not supported for use with Okta Classic APIs.
+
+> **Note:** This doc is designed for people familiar with Okta Classic. If you are new to Okta and Okta Identity Engine, see [Get started](https://help.okta.com/en/oie/okta_help_CSH.htm#ext-get-started-oie) with Okta Identity Engine.
+
+Are you an admin? See the Identity Engine [Limitations](https://help.okta.com/en/oie/okta_help_CSH.htm#ext-oie-limitations) doc for admins.
+
+### Okta Classic features not supported in Okta Identity Engine
+
+#### Event Type availability for Event Hooks
+
+**What Changed:** The following Event Types aren't available in the Identity Engine because Device Trust isn't yet supported:
+
+* `user.authentication.authenticate`
+* `user.credential.enroll`
+
+The following Event Types are available only in the Identity Engine and can't be used by Okta Classic customers:
+
+* `device.enrollment.create`
+* `user.mfa.factor.suspend`
+* `user.mfa.factor.unsuspend`
+* `security.authenticator.lifecycle.activated`
+* `security.authenticator.lifecycle.deactivate`
+
+**Further Information:** [Event Types](/docs/reference/api/event-types/)
+
+***
+
+#### Help Support number
+
+**What Changed:** In Okta Identity Engine, if the user is unable to use an authenticator, rather than providing a Help Support number, the Authenticator list page appears enabling the user to select another way to authenticate.
```

Comment:
```suggestion
**What Changed:** In Okta Identity Engine, if the user is unable to use an authenticator, rather than providing a Help Support number, the Authenticator list page appears that enables the user to select another way to authenticate.
```